guides/v2.1/design-styleguide/errors-and-messages/errors-and-messages.md

```diff
@@ -16,11 +16,11 @@ Magento uses different types of dynamic messaging to alert users when:
 
 ## Types of Errors and Messages
 
-* **Information information.** This messaging is not an error. It's used when a person can continue with the task, but needs to be alerted regarding information critical to completing a task.
+* **Information information.** This messaging is not an error. It's used when a person can continue with the task but needs to be alerted regarding information critical to completing a task.
 
 * **Recoverable error.** Used when a person is blocked and can only continue by taking an action to fix the problem.
 
-* **Not recoverable / catastrophic error.** Used when a person can’t complete the task currently. Usually, it’s due to system error, server being down, etc. For ideal next steps for users, include a contact method to fix the issue. If a contact method isn't available, provide guidance for the user to try again at a later time.
+* **Not recoverable / catastrophic error.** Used when a person can’t complete the task currently. Usually, it’s due to system error, the server is down, etc. For ideal next steps for users, including a contact method to fix the issue. If a contact method isn't available, provide guidance for the user to try again at a later time.
```

Comment:
The second change in this paragraph is not correct. The original test is correct.

> For ideal next steps for users, ~including~ **include** a contact method...